laravel/tests/Unit/ScheduleParserTest.php

```diff
@@ -11,6 +11,16 @@ class ScheduleParserTest extends TestCase
 {
     use DatabaseTransactions;
 
+    // sets a private method for a given class to be accessible so it can be used in testing
+    // link: https://stackoverflow.com/questions/249664/best-practices-to-test-protected-methods-with-phpunit
+    public function getPrivateMethod($className, $methodName)
+    {
+        $reflector = new \ReflectionClass($className);
+        $method = $reflector->getMethod($methodName);
+        $method->setAccessible(true);
+        return $method;
+    }
+
```

Comment:
Given that this now appears in many of our test classes, it might be good to move it to a helper file to avoid duplicated code.